src/dailyai/services/base_transport_service.py

```diff
@@ -64,8 +130,11 @@ async def run(self):
         self._frame_consumer_thread.start()
 
         if self._speaker_enabled:
-            self._receive_audio_thread = threading.Thread(target=self._receive_audio, daemon=True)
-            self._receive_audio_thread.start()
+            # TODO-CB: daily-python doesn't like these two listening at the same time
```

Comment:
It feels like we should be able to write a service that consumes audio frames to do VAD detection in the same way the client-side transcription works.

Reply:
ugh FINE DAD i'll do it the """right way"""